Dockerfile

```diff
@@ -1,15 +1,25 @@
-FROM golang:1.19.8-alpine as builder
```

Comment:
We might as well use the default golang container that is based off of Debian bullseye and has glibc installed, since it matches what our final container will be. This is definitely the way to go if the final container will be alpine, but `monero-wallet-rpc` depends on glibc, so it's not the best option.